resources/conf/config.toml

```diff
@@ -42,13 +55,16 @@ password = "wso2carbon"
     consumerKeyClaim = "azp"
 
 # Configurations for retrieving API and subscription data from API Manager.
-[filter.eventHub]
+[enforcer.eventHub]
     # Enable/ Disable the feature
     enable = false
     # The API Manager URL
     serviceUrl = "https://localhost:9443/"
     # User name and password of the internal data api.
+    [enforcer.eventHub.jmsConnectionParameters]
+        # The message broker connection URL.
+        eventListeningEndpoints = "amqp://admin:admin@carbon/carbon?brokerlist='tcp://localhost:5672'"
+
+[enforcer.apimCredentials]
     username="admin"
-    password="admin"
-    # The message broker connection URL.
-    eventListeningEndpoints = "amqp://admin:admin@carbon/carbon?brokerlist='tcp://localhost:5672'"
+    password=['a','d','m','i','n']
```

Comment:
for the password, shall can't we use a string ?
new line missing

Reply:
I too think we should keep this as a string and convert it to charset at TOML parser level.